internal/monitor_conf_test.go

```diff
@@ -1113,8 +1086,8 @@ var _ = Describe("monitor_conf", func() {
 
 		Context("with a version that can use binaries from the main container", func() {
 			BeforeEach(func() {
-				cluster.Spec.Version = fdbtypes.Versions.WithBinariesFromMainContainer.String()
```

Comment:
Do we still need a special test case for this if it is the only supported behavior?